app/modules/repo_manager/sync_helper.py

```
@@ -239,6 +241,26 @@ def _lookup() -> Optional[str]:
+            asyncio.create_task(
+                email_helper.send_parsing_failure_alert(
+                    repo_name=repo_name,
+                    branch_name=ref,
+                    error_message=f"All authentication methods failed: {str(e)}",
+                    auth_method="all_methods",
+                    failure_type="cloning_auth",
+                    user_id=user_id,
+                    project_id=None,
+                    stack_trace=traceback.format_exc(),
+                )
+            )
```

Ruff (0.15.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)